packages/analytics-js/src/state/slices/consents.ts

```diff
@@ -3,14 +3,14 @@ import { Signal, signal } from '@preact/signals-core';
 export type ConsentsState = {
   deniedConsentIds: Signal<string[]>;
   allowedConsents: Signal<Record<string, string>>;
-  activeConsentProvider: Signal<string | undefined>;
+  activeConsentProviderPluginName: Signal<string | undefined>;
```

Comment:
This signal should be of type `PluginName`, right?

Reply:
Done